tests/app/test_dependencies.py

```diff
@@ -38,6 +44,18 @@
     LiteratureSearchTool,
     MorphologyFeatureTool,
 )
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+
+
+def test_get_settings(monkeypatch):
+    monkeypatch.setenv("NEUROAGENT_TOOLS__LITERATURE__URL", "https://localhost1")
+    monkeypatch.setenv("NEUROAGENT_KNOWLEDGE_GRAPH__BASE_URL", "https://localhost2")
+    monkeypatch.setenv("NEUROAGENT_KEYCLOAK__USERNAME", "fake_username")
+    monkeypatch.setenv("NEUROAGENT_KEYCLOAK__PASSWORD", "fake_password")
```

Comment:
These two variables are not needed anymore, since in the `patch_required_env` fixture we have another authetification method, it should work.